### PR TITLE
fix ecc curve test in normal math mode, free is needed

### PR DESCRIPTION
**Pull request by toddouska**

### CODE REVIEW

**Comment by dgarske** [comment]
Thanks for the fix! Looks good to me. Will merge when Jenkins is done chewing on it.

File: wolfcrypt/test/test.c [hunk 1]
```diff
@@ -6259,6 +6259,7 @@ static int ecc_test_vector_item(const eccVector* vector, WC_RNG* rng)
     memset(sig, 0, sizeof(sig));
     x = sizeof(sig);
 
+    wc_ecc_free(&userA);  /* free key before importing a new one over the top */
```
Comment (dgarske) [inline wolfcrypt/test/test.c]: Looking at this again and wc_ecc_make_key and wc_ecc_check_key do not need to be here. Which also means this new wc_ecc_free doesn't need to be here. Lines 6251 through 6262 should just be removed.

Comment (toddouska) [inline wolfcrypt/test/test.c]: That's another way to fix it for sure!  I was thinking maybe the make key / check key pair was for testing different sizes.  Maybe not?

**Comment by toddouska** [comment]
John, please review.  No one who commited to the PR should merge it.

**Pull request merged**